frontend/app/actions/Fallbacks.scala

```diff
@@ -1,11 +1,48 @@
 package actions
 
+import com.gu.memsub.subsv2.CatalogPlan.PaidMember
+import com.gu.memsub.subsv2.MonthYearPlans
 import com.gu.salesforce.PaidTier
 import play.api.mvc.Results._
-import play.api.mvc.{Call, RequestHeader}
+import play.api.mvc.{Call, Cookie, RequestHeader}
 import play.twirl.api.Html
 import configuration.Config
+import views.html.joiner.form.paymentB
+import views.support.{CountryWithCurrency, IdentityUser, PageInfo}
 
+import scala.util.Random
+
+
+sealed trait CheckoutFlowVariant {
+  val testId: String
+  val identitySkin: String
+}
+
+object CheckoutFlowVariant {
```

Comment:
These `CheckoutFlowVariant ` classes probably don't belong in `package actions`. Perhaps use IntelliJ to move them to a new `abtests` package?